upload-release.sh

```diff
@@ -17,7 +22,7 @@ if [ -z "$BOSH_PASSWORD" ]; then
   exit 1
 fi
 
-bosh --ca-cert certificate/boshCA.crt -n target $BOSH_TARGET
+bosh --ca-cert $BOSH_CERT -n target $BOSH_TARGET
```

Comment:
If `certificate` is still a required input, why not just do `certificate/$BOSH_CERT` so you only have to supply filename.